### PR TITLE
[MWPW-154906] Pricing Cards Credits For AI Pages

### PR DESCRIPTION
**Pull request by echen-adobe**
Describe your specific features or fixes

Implements a new pricing cards block for AI pages. This card is a stripped down version of our pricing-cards that shows the number of free AI tokens a user can expect

Screenshot 2024-07-30 at 3 41 06 PM

Resolves:  https://jira.corp.adobe.com/browse/MWPW-154906

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://ai-page-pricing--express--adobecom.hlx.page/drafts/echen/pricing-cards-credits

### CODE REVIEW

**Review by fullcolorcoder — APPROVED** [review]

File: test/unit/blocks/pricing-cards-credits/pricing-cards-credits.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  });
+
+  it('Contains pricing-area-wrapper elements', () => {
+    const pricingAreaWrappers = pricingCardsCredits.querySelectorAll('.pricing-area-wrapper');
+    expect(pricingAreaWrappers).to.have.length(2);
+  });
+
+  it('Contains compare-all class on button containers', () => {
+    const compareAllButtons = pricingCardsCredits.querySelectorAll('.compare-all');
+    expect(compareAllButtons.length === 2).to.be.true;
```
Comment (kennethlum) [inline test/unit/blocks/pricing-cards-credits/pricing-cards-credits.test.js:52]: would this also be possible be in the form like line 47 above?

File: express/blocks/pricing-cards-credits/pricing-cards-credits.css [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 20px;
+}
+
+.pricing-cards-credits .card .compare-all>a {
+  color: var(--color-info-accent);
+  border: none;
+  background: none;
+  padding: 0px;
```
Comment (kennethlum) [inline express/blocks/pricing-cards-credits/pricing-cards-credits.css:148]: can be `0`?

File: express/blocks/pricing-cards-credits/pricing-cards-credits.css [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+  text-decoration: underline;
+  font-weight: 400;
+  margin-top: 15px;
+  font-size: 14pt;
+  margin-bottom: 0px;
```
Comment (kennethlum) [inline express/blocks/pricing-cards-credits/pricing-cards-credits.css:153]: can be `0`?

File: express/blocks/pricing-cards-credits/pricing-cards-credits.css [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+  .pricing-cards-credits .card {
+    width: calc(100% - 40px);
+    margin: auto;
+    min-height: unset;
+  }
+
+  .pricing-cards-credits .card-feature-list {
+    display: none;
+    min-height: 0px;
```
Comment (kennethlum) [inline express/blocks/pricing-cards-credits/pricing-cards-credits.css:172]: can be `0`?

File: express/blocks/pricing-cards-credits/pricing-cards-credits.css [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+  font-size: 14pt;
+  margin-bottom: 0px;
+  text-align: center;
+}
+
+@media (max-width: 600px) {
+  .pricing-cards-credits {
+    flex-wrap: wrap;
+    gap: 10px;
+    min-width: none;
```
Comment (JingleH) [inline express/blocks/pricing-cards-credits/pricing-cards-credits.css:161]: i think unset might get you what you want here?

**Review by JingleH — CHANGES_REQUESTED** [review]

**Review by JingleH — APPROVED** [review]

**Pull request merged**